src/components/elements/MapPolygonPanel/AttributeInformation.tsx

```diff
@@ -156,11 +174,22 @@ const AttributeInformation = () => {
         num_trees: treesPlanted
       };
       try {
-        await fetchPutV2TerrafundSitePolygonUuid({
-          body: updatedPolygonData,
-          pathParams: { uuid: polygonData.uuid }
-        });
-        setShouldRefetchPolygonData(true);
+        sendSiteData(
+          {
+            body: updatedPolygonData,
+            pathParams: { uuid: polygonData.uuid }
+          },
+          {
+            onSuccess: () => {
+              setShouldRefetchPolygonData(true);
+              setEditPolygon({ isOpen: false, uuid: "" });
+              displayNotification(t("Polygon data updated successfully"), "success", "Success!");
+            },
+            onError: error => {
+              displayNotification(t("Error updating polygon data"), "error", "Error!");
```

Comment:
`Success!` on line 186 and `Error!` here need translation

Reply:
done